{"name": "test_auth[sanya_2k@mail.ru-Strider2006]", "status": "failed", "statusDetails": {"message": "AssertionError: assert not True\n +  where True = <bound method WebElement.is_visible of <pages.elements.WebElement object at 0x000002337EBF3DC8>>()\n +    where <bound method WebElement.is_visible of <pages.elements.WebElement object at 0x000002337EBF3DC8>> = <pages.elements.WebElement object at 0x000002337EBF3DC8>.is_visible\n +      where <pages.elements.WebElement object at 0x000002337EBF3DC8> = <pages.pages_auth.RTmis object at 0x000002337F71DEC8>.error", "trace": "web_browser = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"ae7b55c315156bee2be355382980a791\")>, login = 'sanya_2k@mail.ru', password = 'Strider2006'\n\n    @allure.feature('Авторизация на https://rtmis.ru/')\n    @allure.story('Тест с авторизации с рандомизируемыми данными')\n    def test_auth(web_browser, login:str, password:str):\n        web = RTmis(web_browser)\n    \n        with allure.step(f\"Открываем {web.url}\"):\n          web.get(url=web.url)\n        with allure.step(\"Заполняем форму (логин:пароль)\"):\n          assert web.login.send_keys(login)\n          assert web.password.send_keys(password)\n        with allure.step(\"Нажимаем на кнопку войти\"):\n          web.auth.human_click()\n        with allure.step(\"Определям на странице сообщение об ошибке авторизации\"):\n>           assert not web.error.is_visible()\nE           assert not True\nE            +  where True = <bound method WebElement.is_visible of <pages.elements.WebElement object at 0x000002337EBF3DC8>>()\nE            +    where <bound method WebElement.is_visible of <pages.elements.WebElement object at 0x000002337EBF3DC8>> = <pages.elements.WebElement object at 0x000002337EBF3DC8>.is_visible\nE            +      where <pages.elements.WebElement object at 0x000002337EBF3DC8> = <pages.pages_auth.RTmis object at 0x000002337F71DEC8>.error\n\ntests\\test_auth.py:18: AssertionError"}, "steps": [{"name": "Открываем https://demo.rtmis.ru/", "status": "passed", "start": 1643733565414, "stop": 1643733568918}, {"name": "Заполняем форму (логин:пароль)", "status": "passed", "start": 1643733568918, "stop": 1643733569051}, {"name": "Нажимаем на кнопку войти", "status": "passed", "start": 1643733569051, "stop": 1643733579336}, {"name": "Определям на странице сообщение об ошибке авторизации", "status": "failed", "statusDetails": {"message": "AssertionError: assert not True\n +  where True = <bound method WebElement.is_visible of <pages.elements.WebElement object at 0x000002337EBF3DC8>>()\n +    where <bound method WebElement.is_visible of <pages.elements.WebElement object at 0x000002337EBF3DC8>> = <pages.elements.WebElement object at 0x000002337EBF3DC8>.is_visible\n +      where <pages.elements.WebElement object at 0x000002337EBF3DC8> = <pages.pages_auth.RTmis object at 0x000002337F71DEC8>.error\n", "trace": "  File \"C:\\Users\\ole lukoie\\Desktop\\ui\\tests\\test_auth.py\", line 18, in test_auth\n    assert not web.error.is_visible()\n"}, "start": 1643733579336, "stop": 1643733579380}], "parameters": [{"name": "login", "value": "'sanya_2k@mail.ru'"}, {"name": "password", "value": "'Strider2006'"}], "start": 1643733565414, "stop": 1643733579380, "uuid": "7405c58a-746c-4a17-8334-a8de6582b4c6", "historyId": "b9075073059464f6d82b127a00d1fb98", "testCaseId": "5306df55395c93b83e69abe887b683bd", "fullName": "tests.test_auth#test_auth", "labels": [{"name": "feature", "value": "Авторизация на https://rtmis.ru/"}, {"name": "story", "value": "Тест с авторизации с рандомизируемыми данными"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_auth"}, {"name": "host", "value": "DESKTOP-2HIEP5G"}, {"name": "thread", "value": "1520-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_auth"}]}